tutorial/README.md

```diff
@@ -0,0 +1,281 @@
+* Add the following to the Coin contract
+```
+contract Coin {
+	//...
+
+    string public name = "Coin";
+    uint256 public decimals = 18;
+    string public symbol = "COIN";
+
+	// Returns balance of an address
+    function balanceOf(address addr) public view returns (uint) {
+        return balances[addr];
+    }
+
+```
+* lastly rename the `function send(` to `function transfer(`
+
+* (optional) you're welcome to modify this Coin example, or use an erc20 contract of your choosing.
+
+
+Once you're happy with your contract compiling and running here, we will next deploy to the Ropsten Ethereum test network.
+
+### Deploy Token Contract
+
+At the time of writing, studio.ethereum.org defaults contract deployment to the JavaScriptVM.
+[Remix](remix.ethereum.org) is a convenient way to deploy to Ropsten:
+
+* Open [remix](remix.ethereum.org) in a new tab, and choose the `Solidity` environment
+* Create a new file (also name it Coin.sol)
+* From studio, copy the source of `contracts/Coin.sol`. Then in remix, paste in the new Coin.sol
+* In the next studio tab, `compiler`, click `Compile Coin.sol`
+* In the next studio tab, click `Deploy`
+  * keep studio open you'll need it later for the contract abi
+* Once deployed, copy the contract address to the clipboard
+
+### View in AlphaWallet
+
+To see the default rendering of the contract as a fungible token:
+
+* Open AlphaWallet on your mobile
+* Tap the `+` in the top-right corner
+  * this is required to manually show tokens that you don't have a balance for
+  * (alternately, in remix you can use the `mint` function, to give your wallet address some Coin, eg: `wallet_address, 88000000000000000000`)
+* Enter your contract address:
+  * The most convenient way is (from your computer) to paste the contract address into any [QR code generator](https://www.cssscript.com/demo/flexible-client-side-qr-code-generator/)
+  * Tap the camera on the address field to scan
+* Once you've added the contract address (QR code or otherwise), you should see the token card `Coin (COIN)`, tagged as `Ropsten`
+
+## Your first TokenScript
+
+### Constructing your xml file (current schema 2019/10)
+
+* In studio, create a new file Coin.xml, and initialise with this template:
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<ts:token xmlns:ts="http://tokenscript.org/2019/10/tokenscript"
+          xmlns="http://www.w3.org/1999/xhtml"
+          xmlns:xml="http://www.w3.org/XML/1998/namespace"
+          xsi:schemaLocation="http://tokenscript.org/2019/10/tokenscript http://tokenscript.org/2019/10/tokenscript/tokenscript.xsd"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          custodian="false"
+>
+    <ts:name>
+        <ts:string xml:lang="en">Coin</ts:string>
+    </ts:name>
+    <ts:contract name="Coin" interface="erc20">
+        <ts:address network="3">YOUR_CONTRACT_ADDRESS</ts:address>
+    </ts:contract>
+    <ts:origins>
+        <ts:ethereum contract="Coin"></ts:ethereum>
```

Comment:
Should mention that this is defining what token the TokenScript is defined for

Reply:
Will add a clarifying note. I had presumed the context was clear.